systemd/coreos-installer-reboot-loaddev.service

```
@@ -0,0 +1,14 @@
+[Unit]
+Description=Reboot from zVM LOADDEV after CoreOS Installer
+ConditionPathExists=/run/coreos-installer-reboot
+ConditionKernelCommandLine=coreos.inst.secure_ipl
+Requires=coreos-installer.target
+After=coreos-installer.target
+OnFailure=emergency.target
+OnFailureJobMode=replace-irreversibly
+
+[Service]
+Type=simple
+ExecStart=/bin/bash -euo pipefail -c '[[ -e /dev/vmcp ]] && vmcp ipl loaddev || systemctl --no-block reboot'
```

The strict mode switches are not useful here.

```suggestion
ExecStart=/bin/sh -c '[ -e /dev/vmcp ] && vmcp ipl loaddev || systemctl --no-block reboot'
```